getcontext/tracing/_tools.py

```diff
@@ -47,6 +49,57 @@ def __user_function_wrapper(func, *args, **kwargs):
     return trace
 
 
+def dynamic_traceable(func: Callable,
+                      run_type: ls_client.RUN_TYPE_T = 'chain',
+                      name: str = None) -> Callable:
+    """
+    Dynamically create a traceable function.
+    
+    Useful when you want to dynamically name a trace or you don't have access to the function definition (e.g. openai SDK functions)
+
+    Args:
+        func (Callable): The function to be traced.
+        run_type (ls_client.RUN_TYPE_T, optional): The type of tracing to be applied. Defaults to 'chain'.
+        name (str, optional): The name of the trace. Defaults to function name.
+
+    Returns:
+        Callable: The wrapped function.
+        
+    Example:
+    .. code-block:: python
+        import openai
+        from getcontext.tracing import dynamic_traceable, capture_trace, Evaluator
+        
+        openai_client = openai.Client()
+        openai_chat = dynamic_traceable(
+            openai_client.chat.completions.create,
+            run_type='llm',
+            name='openai_hello_world')
+        
+        trace = capture_trace(
+            openai_chat,
+            messages=[{"role": "user", "content": "Tell me a fun fact about the world."}],
+            model="gpt-3.5-turbo")
+            
+        evaluator = Evaluator('golden_response', {'golden_response': 'Hello, world'})
```

Comment:
I think Evaluator needs the keyword argument names here:

```
evaluator = Evaluator(evaluator='golden_response', options={'golden_response': 'Hello, world'})
```

Reply:
good catch, done